ui/lib/config-ui/addon/routes/login-settings/rule/details.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+
+export default class LoginSettingsRuleDetailsRoute extends Route {
+  @service('app-router') router;
+  @service api;
+
+  beforeModel() {
+    const { name } = this.paramsFor('login-settings.rule');
+    if (!name) {
+      this.router.transitionTo('vault.cluster.config-ui.login-settings.index');
+    }
+  }
+
+  async model() {
+    const { name } = this.paramsFor('login-settings.rule');
+
+    const rule = await this.api.sys.uiLoginDefaultAuthReadConfiguration(name);
+
+    return { rule: { name, ...rule.data } };
+  }
+
+  setupController(controller, resolvedModel) {
+    super.setupController(controller, resolvedModel);
+
+    controller.breadcrumbs = [{ label: 'UI login rules', route: 'login-settings' }, { label: '' }];
```

Comment:
The label here should be the rule name (without the route, like you have it here) - sometimes the designs don't show the most up-to-date breadcrumbs 😅 
```suggestion
    controller.breadcrumbs = [{ label: 'UI login rules', route: 'login-settings' }, { label: resolvedModel.rule.name }];
```